internal/config/types/sql.go

```diff
@@ -68,16 +70,21 @@ func (s *SQL) connectionString() string {
 		"application_name=zitadel",
 		"sslmode=" + s.SSL.Mode,
 	}
+	if s.ClusterName != "" {
+		fields = append(fields, "options=--cluster="+s.ClusterName)
```

Comment:
wouldn't it make sense to add the Field `Options map[string]string` where you can add any option you want? because if we want to add another option we had to change the code